buffer/src/main/java/io/netty/buffer/PooledByteBufAllocator.java

```diff
@@ -103,6 +104,8 @@
         DEFAULT_CACHE_TRIM_INTERVAL = SystemPropertyUtil.getInt(
                 "io.netty.allocator.cacheTrimInterval", 8192);
 
+        DEFAULT_MAX_POOLED_BYTES = SystemPropertyUtil.getLong("io.netty.allocator.maxPooledBytes", Long.MAX_VALUE);
```

Comment:
I'm for io.netty.allocator.maxPooledBytes if it represents max pooled bytes for all arenas.